.eslintrc.js

```diff
@@ -269,6 +269,7 @@ module.exports = {
       excludedFiles: [
         'app/scripts/controllers/app-state.test.js',
         'app/scripts/controllers/mmi-controller.test.js',
+        'app/scripts/controllers/metametrics.test.js',
```

Comment:
My understanding for this section is, we will exclude lint rules for some `Mocha tests`. since this one has been migrated, we should not add more to it?

Reply:
These are the `excludedFiles`, i.e. the files that should be excluded from the mocka lint rules

Reply:
Yes, but if this file has been converted to jest, is mocha still detecting and enforcing rules ?